src/GameStateMatch.cpp

```diff
@@ -209,11 +217,10 @@ void GameStateMatch::render() {
 }
 
 GameStateMatch::~GameStateMatch() {
-
     // Free texture and font
     delete GameManager::instance();
     frameFPSTextTexture.free();
     TTF_CloseFont(frameFont);
-    frameFont = NULL;
+    frameFont = nullptr;
```

Comment:
Setting pointer to null inside destructor is pointless as it will be deleted, so no references will exist
Remove line